templates/docs/examples/brochure/_25-25-50.html

```diff
@@ -1,6 +1,6 @@
 <section class="p-section">
   <div class="row">
-    <hr class="p-rule">
```

Comment:
Let's not remove these. With the update of CSS they work the same with or without class name, but it's safer to keep the class name (for future compatibility).